pkg/sdk/file_format_test.go

```diff
@@ -168,22 +144,18 @@ func TestFileFormatsCreate(t *testing.T) {
 			Type:        FileFormatTypeXML,
 
 			FileFormatTypeOptions: FileFormatTypeOptions{
-				XMLCompression:              &XMLCompressionZstd,
-				XMLIgnoreUTF8Errors:         Bool(true),
-				XMLPreserveSpace:            Bool(true),
-				XMLStripOuterElement:        Bool(true),
-				XMLDisableSnowflakeData:     Bool(true),
-				XMLDisableAutoConvert:       Bool(true),
-				XMLReplaceInvalidCharacters: Bool(true),
```

Comment:
why was this removed ?

Reply:
Because this is not valid configuration (cant be used with XMLIgnoreUTF8Errors)